### PR TITLE
Much more frequent imports

### PR DESCRIPTION
**Pull request by another-rex**
Remove limitation of once a day imports

### CODE REVIEW

File: docker/importer/importer.py [hunk 1]
```diff
@@ -174,10 +174,6 @@ def import_new_oss_fuzz_entries(self, repo, oss_fuzz_source):
 
   def schedule_regular_updates(self, repo, source_repo):
     """Schedule regular updates."""
-    if (source_repo.last_update_date and
```
Comment (oliverchang) [inline docker/importer/importer.py:177]: This was the only place this property was being used. Let's just get rid of it completely (from models.py too).

**Review by oliverchang — COMMENTED** [review]
thanks!

File: osv/impact.py [hunk 2]
```diff
@@ -91,7 +91,9 @@ def get_affected(self,
                    fix_commits,
                    limit_commits=None):
     """"Get list of affected tags and commits for a bug given regressed and
-    fixed commits."""
+
```
Comment (oliverchang) [inline osv/impact.py:94]: unintentional change?

File: osv/impact.py [hunk 3]
```diff
@@ -196,8 +198,11 @@ def _get_equivalent_commit(self,
                              to_search,
                              target_commit,
                              detect_cherrypicks=True):
-    """Find an equivalent commit at to_search, or None. The equivalent commit
-    can be equal to target_commit."""
+    """Find an equivalent commit at to_search, or None.
+
+    The equivalent commit
```
Comment (oliverchang) [inline osv/impact.py:203]: unintentional change?

File: osv/semver_index.py [hunk 4]
```diff
@@ -64,7 +64,9 @@ def parse(version):
 
 def normalize(version):
   """Normalize semver version for indexing (to allow for lexical
-  sorting/filtering)."""
+
+  sorting/filtering).
```
Comment (oliverchang) [inline osv/semver_index.py:68]: unintentional change?

File: osv/vulnerability_pb2.py [hunk 5]
```diff
@@ -10,740 +10,1203 @@
```
Comment (oliverchang) [inline osv/vulnerability_pb2.py:10]: was this meant to be changed here?

File: osv/third_party/univers/gem.py [hunk 6]
```diff
@@ -21,15 +21,15 @@
 
 
 class InvalidRequirementError(AttributeError):
-    pass
```
Comment (oliverchang) [inline osv/third_party/univers/gem.py:24]: no need to format third_party code.

File: docker/importer/importer.py (hunk 1 above)
Comment (oliverchang) [inline docker/importer/importer.py:177]: Sorry, I completely forgot about this case where we still need it. Let's revert to your old way for now.

File: osv/impact.py (hunk 2 above)
Comment (another-rex) [reply osv/impact.py:94]: Ah... I set up a git hook to run pyformat on all files committed. Not sure why pyformat likes adding blank lines to multiline doc comments though

File: docker/importer/importer.py (hunk 1 above)
Comment (oliverchang) [reply docker/importer/importer.py:177]: The condition we want to delete is the one in  `_process_updates_bucket`. This one is the the one we need to keep.

Comment (another-rex) [reply docker/importer/importer.py]: Ah ok, should be fixed now

**Review by oliverchang — APPROVED** [review]

**Pull request merged**